mne/transforms.py

```diff
@@ -26,25 +26,80 @@
 als_ras_trans_mm = als_ras_trans * [0.001, 0.001, 0.001, 1]
 
 
+_str_to_frame = dict(meg=FIFF.FIFFV_COORD_DEVICE,
+                     mri=FIFF.FIFFV_COORD_MRI,
+                     mri_voxel=FIFF.FIFFV_MNE_COORD_MRI_VOXEL,
+                     head=FIFF.FIFFV_COORD_HEAD,
+                     mni_tal=FIFF.FIFFV_MNE_COORD_MNI_TAL,
+                     ras=FIFF.FIFFV_MNE_COORD_RAS,
+                     fs_tal=FIFF.FIFFV_MNE_COORD_FS_TAL,
+                     ctf_head=FIFF.FIFFV_MNE_COORD_CTF_HEAD,
```

Comment:
@dengemann we could call this `c4k_head` (CTF/4D/Kit) or `nnm_head` (non-neuromag) instead...? That might make it clearer in Python that it really means a non-neuromag system.